### ISSUE TITLE
Delete the CHANGELOG.md

### ISSUE THREAD

**Issue by jimmywarting** [node-fetch/node-fetch#1372]
@node-fetch/core what do you think about ditching the CHANGELOG.md?

Everytime we release a new tag, then we have the possibility to auto generate a changelog from all PR/commits that where added. 
that way we don't have to manually keep the changelog up to date.

**Comment by jimmywarting** [comment]
Here is an example from latest release: https://github.com/node-fetch/node-fetch/releases/tag/v3.1.0

**Comment by LinusU** [comment]
I'm 👍  on this

This is how I do it in all of my repositories as well

**Comment by gr2m** [comment]
Once we merge https://github.com/node-fetch/node-fetch/pull/1270, releases with changelogs will be auto-generated. I'm okay with deleting the `CHANGELOG.md` file.

**Comment by ramya-rao-a** [comment]
One catch here is that the generated changelog in the releases does not categorize the changes into breaking changes vs new features vs bug fixes. When there is a major version update, one of the things important for consumers would be to understand what the breaking changes are

**Comment by gr2m** [comment]
That's not right, semantic-release does separate bug fixes, features, and breaking changes in the release notes it generates.

**Comment by ramya-rao-a** [comment]
Apologies, I did not see such break up in https://github.com/node-fetch/node-fetch/releases/tag/v3.0.0 and so the comment. I realize now that the v3 release might have been before adapting the auto generated changelog

**Comment by gr2m** [comment]
we don't use semantic-release yet, #1270 is pending
